src/options.ts

```diff
@@ -46,6 +51,7 @@ export interface Options {
   tracerConfig: NodeTracerConfig;
   spanExporterFactory: SpanExporterFactory;
   spanProcessorFactory: SpanProcessorFactory;
+  propagatorFactory: PropagatorFactory;
```

Comment:
I don't imagine propagators will need to be initialized lazily or will have any dependency on config or other components but for the sake of consistency I followed factory pattern.